frontend/src/components/VSkeleton/VGridSkeleton.vue

```diff
@@ -44,19 +44,23 @@ export default defineComponent({
     },
     numElems: {
       type: Number,
-      default: function () {
-        if (this.isForTab === "all") return 20
-        if (this.isForTab === "image") return 30
-        return 8
-      },
+      default: 0,
```

Comment:
```suggestion
```
Default here means that if the prop is not given, the default will be used. So, the function you put inside setup will never go past `if prop.numElems`: prop.numElems will always be defined.
Removing the default will make it so that numElems is undefined, and will be computed inside setup.